dss/operations/params.py

```diff
@@ -0,0 +1,146 @@
+"""
+Get, set, and unset environment variables in the SSM store and in deployed lambda functions
+"""
+import boto3
+import os
+import typing
+import argparse
+import json
+import logging
+
+from dss.operations import dispatch
+
+
+logger = logging.getLogger(__name__)
+
+
+def get_ssm_lambda_environment():
+    ssm_client = boto3.client("ssm")
+    parms = ssm_client.get_parameter(
+        Name=f"/{os.environ['DSS_PARAMETER_STORE']}/{os.environ['DSS_DEPLOYMENT_STAGE']}/environment"
+    )['Parameter']['Value']
+    return json.loads(parms)
+
+def set_ssm_lambda_environment(parms: dict):
+    ssm_client = boto3.client("ssm")
+    ssm_client.put_parameter(
+        Name=f"/dcp/dss/{os.environ['DSS_DEPLOYMENT_STAGE']}/environment",
+        Value=json.dumps(parms),
+        Type="String",
+        Overwrite=True,
+    )
+
+def get_deployed_lambda_environment(name):
+    lambda_client = boto3.client("lambda")
+    return lambda_client.get_function_configuration(FunctionName=name)['Environment']['Variables']
+
+def set_deployed_lambda_environment(name, env: dict):
+    lambda_client = boto3.client("lambda")
```

Comment:
lets upgrade this to `dss.util.aws.clients`

Reply:
This comment has been addressed